components/test-suite/src/loci/tests/testng/Configuration.java

```diff
@@ -104,7 +104,14 @@
   private static final String X_POSITION = "PositionX_";
   private static final String Y_POSITION = "PositionY_";
   private static final String Z_POSITION = "PositionZ_";
-
+  private static final String TIME_INCREMENT_UNITS = "TimeIncrementUnits";
```

Comment:
Is there a reason to pluralize these keys since there is only one unit per metadata type?